Orange/ensembles/ada_boost.py

```diff
@@ -1,24 +1,37 @@
 import sklearn.ensemble as skl_ensemble
-from Orange.base import SklLearner
-from Orange.classification.base_classification import (SklLearnerClassification,
-                                                       SklModelClassification)
-from Orange.regression.base_regression import (SklLearnerRegression,
-                                               SklModelRegression)
 
-__all__ = ["SklAdaBoostLearner", "SklAdaBoostRegressionLearner"]
+from Orange.base import SklLearner, SklModel
+from Orange.classification.base_classification import (
+    SklLearnerClassification, SklModelClassification
+)
+from Orange.modelling import Fitter
+from Orange.regression.base_regression import (
+    SklLearnerRegression, SklModelRegression
+)
+
+__all__ = [
+    'SklAdaBoostLearner',
+    'SklAdaBoosClassificationLearner',
+    'SklAdaBoostRegressionLearner',
+]
 
 
 class SklAdaBoostClassifier(SklModelClassification):
     pass
 
 
-class SklAdaBoostLearner(SklLearnerClassification):
+class SklAdaBoosClassificationLearner(SklLearnerClassification):
     __wraps__ = skl_ensemble.AdaBoostClassifier
     __returns__ = SklAdaBoostClassifier
-    name = 'skl adaBoost'
+    name = 'skl adaBoost classification'
```

Comment:
It's very clever, but since it's a property on the class, it will only work as expected when dealing with an instance. It doesn't actually work if you tried to do `SomeLearner.name` which happens [here](https://github.com/biolab/orange3/pull/1819/files#diff-ac6f5b024d3d33a616daa15703ec0a64R21). This is the only instance where I've come across this, and it's very easily fixable here. Just an observation.